Project.toml

```diff
@@ -59,7 +59,7 @@ UnitfulExt = "Unitful"
 Aqua = "0.8"
 Contour = "0.5 - 0.6"
 Downloads = "1"
-FFMPEG = "0.3, 0.4"
+FFMPEG = "0.3, 0.4, 0.4.1"
```

Comment:
```suggestion
FFMPEG = "0.3, 0.4.1"
```

A compat of 0.4 already includes 0.4.1, but we can try to force people to get 0.4.1 by excluding 0.4.0

Reply:
Sorry, I couldn't respond earlier. That's strange. Ideally, it should try to install 0.4.1 since it's available. But when I tried to `add Plots`, it tried to install FFMPEG 0.4 and failed.

![Screenshot 2024-09-12 at 3 25 39 PM](https://github.com/user-attachments/assets/6d720b80-5631-4d0a-925d-5b8280973a42)

Reply:
The ways of the resolver are mysterious. You can try explicitly requiring FFMPEG 0.4.1 either by setting a compat in your Project file or by adding it via `] add FFMPEG@0.4.1`